.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+
+      - name: Install purescript-psa
+        run: npm install -g purescript-psa
+
+      - uses: thomashoneyman/setup-purescript@v1
+        with:
+          purescript: "0.13.8"
```

Comment:
With this, we could technically remove the `purescript` and `spago` dependencies from `package.json` as they aren't needed in CI. But I kept them there for now for local development in case folks want them.